maskrcnn_benchmark/engine/inference.py

```diff
@@ -366,7 +363,9 @@ def inference(
     )
     logger = logging.getLogger("maskrcnn_benchmark.inference")
     dataset = data_loader.dataset
-    logger.info("Start evaluation on {} images".format(len(dataset)))
+    assert hasattr(dataset, 'name'), 'Dataset must has a name to perform evaluating.'
+    dataset_name = dataset.name
```

Comment:
I was actually thinking about passing a `dataset_name` as an argument to `inference`, instead of adding a field in the dataset itself. What do you think?